shared/settings/account/container.tsx

```diff
@@ -8,16 +8,38 @@ import AccountSettings from '.'
 import {isMobile} from '../../styles'
 
 type OwnProps = {}
-const mapStateToProps = (state: TypedState) => ({
-  _emails: state.settings.email.emails,
-  _phones: state.settings.phoneNumbers.phones,
-  addedEmail: state.settings.email.addedEmail,
-  bootstrapDone: state.settings.email.emails !== null && state.settings.phoneNumbers.phones !== null,
-  hasPassword: !state.settings.password.randomPW,
-  waiting: anyWaiting(state, Constants.loadSettingsWaitingKey),
-})
+const mapStateToProps = (state: TypedState) => {
+  const phoneNumberRecord =
+    state.settings.phoneNumbers.phones &&
+    state.settings.phoneNumbers.phones.find(phoneNumber => phoneNumber.superseded)
+
+  return {
+    _emails: state.settings.email.emails,
+    _phones: state.settings.phoneNumbers.phones,
+    _supersededPhoneNumberKey: phoneNumberRecord && phoneNumberRecord.e164,
+    addedEmail: state.settings.email.addedEmail,
+    bootstrapDone: state.settings.email.emails !== null && state.settings.phoneNumbers.phones !== null,
+    hasPassword: !state.settings.password.randomPW,
+    supersededPhoneNumber: phoneNumberRecord
+      ? {
```

Comment:
can you cut out those of these fields you don't need?

Reply:
i just realized that after all my changes i just needed 1 field